Makefile

```diff
@@ -15,6 +22,7 @@ lint: ## run the fast go linters
 .PHONY: lint
 
 deps:
+	go get -u ./...
```

Comment:
I don't think we want the `-u` (upgrade) here since `deps` is run regularly as a prerequisite for other targets.